docs/en/connector-v2/sink/Maxcompute.md

```diff
@@ -0,0 +1,80 @@
+# Maxcompute
+
+> Maxcompute sink connector
+
+## Description
+
+Used to read data from Maxcompute.
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
```

Comment:
Sink Connector have no `Key features` named `batch` and it only have two `Key features`

<img width="1251" alt="image" src="https://user-images.githubusercontent.com/32193458/205663740-18d4e63a-d172-4e48-8f54-ba1f30ce9b50.png">